### DISCUSSION TITLE
lens_model_DESIJ2318-1106

### DISCUSSION THREAD

**Discussion by JobairHasan** [AstroBridge/BDLensing#226] [General]
I had been assigned as the manager of the model and trying different initial fittings since then. Initially, the bright, round blob on the left seemed to be the perfect counter-image of the main lensed arc (right). The models could reconstruct the general shape of the counter image with large residuals over the whole blob. Then in a telecon, @ZareefJafar pointed out that the blob is too round to be a counter image and might be a star, which seemed very plausible. @ajshajib suggested using a faint blob with an arrowhead shaped residual, as the counter-image which is much closer to the central lensing galaxy than the previous one, This is the counter-image which I used in the subsequent model fittings. Though the reduced Chi-squared value was as good as 1.24, the models couldn't reconstruct the counter-image. What these instead did was shift the mass further right to project the counter-image on the central lensing galaxy. I was very anxious not to constrain the light and the mass distributions of the main lensing galaxy and thus couldn't comprehend the reason for this anomaly. @SulymanHossain pointed out that it might get fixed if I was to constrain the light and the mass distributions. First, I ran a model with the constraint and then again with the initialization at the result of the last run but with the constraint removed. The problem of shifting mass and light centers appears to be fixed, yet no improvement on the counter-image and its periphery is visible. Meanwhile, I have also tried adjusting gamma and beta values with no significant result. Only the adjustment of fixed parameters is left which I think would not be fruitful either. I cannot see any scope of improvement in the existing model. @ajshajib initially suggested that the model might need to include multiple source planes (with different redshifts). What should my next step be?
![__results___12_1](https://github.com/ajshajib/BDLensing/assets/21108350/01240a2a-d165-407a-ad81-cb0989b6beb3)
![__results___28_2](https://github.com/ajshajib/BDLensing/assets/21108350/4cf43040-8c38-414c-94c7-5d6ef88a18d4)
![__results___29_1](https://github.com/ajshajib/BDLensing/assets/21108350/8b423f51-e246-4f9c-b2fe-78e29a5c43c5)

**Comment by SulymanHossain** [comment]
From the Source Light plot, it seems that the model is creating the counter object at the lens galaxy center, whereas it should be away from the center according to the Observed Light plot. So, this system needs a higher `theta_E` value than the current best-fit value. Have you tried setting the lower limit of `theta_E` greater than your current best-fit value of `theta_E`?

**Reply by JobairHasan** [reply]
I have tried this more than once without any success.

**Reply by SulymanHossain** [reply]
> I have tried this more than once without any success.

@JobairHasan bhai, can you please share your latest notebook?
